Assets/_BForBoss/_Core/Scripts/Labor/RingSystem/RingLaborManager.cs

```diff
@@ -26,8 +28,9 @@ public void Reset()
             _laborSystem = new LaborSystem(_ringSystems);
         }
         
-        public void Initialize()
+        public void Initialize(Action<bool> onLaborCompleted)
```

Comment:
just a btw, you don't have you put it in the init if u always update it.

you can just make it a `public Action<Bool>` and update it whenever needed.

Reply:
Actually, I didn't have to keep updating `onLaborCompleted` and I only need to initialize the RingLabor once, right as we leave the Tutorial phase and enter the First Phase. Every phase change after, we just need to reset the RingLabor system